cfme/infrastructure/config_management.py

```diff
@@ -360,9 +360,7 @@ def delete(self, cancel=False, wait_deleted=True, force=False):
 
     @property
     def _refresh_flash_msg(self):
-        return version.pick({'5.7': 'Refresh Provider initiated for 1 provider ({})'.
-                                    format(self.type),
-                             '5.8': 'Refresh Provider initiated for 1 provider'})
+        return 'Refresh Provider initiated for 1 provider'
```

Comment:
This can probably just be a class attribute.

Reply:
OK

Reply:
Done.